src/files.js

```diff
@@ -149,6 +149,7 @@ FOAM_FILES([
   { name: "foam/util/EmailTest" },
   { name: "foam/util/PasswordTest" },
   { name: "foam/util/SecurityUtilTest" },
+  { name: "foam/util/PasswordStrengthGauge" },  
```

Comment:
This is in package foam.util but you require it below as just foam.PasswordStrengthGauge. In either case, we should find a better package than just foam/util.